root/curs/front_page_quick_links.mhtml

```diff
@@ -10,13 +10,22 @@
   <div class="curs-box-body">
     <ul class="curs-quick-links-list">
 % for my $annotation_type (@annotation_type_list) {
+%   my $feature_subtype = $annotation_type->{feature_subtype} || 'NONE';
+%   my $show_link = ($feature_subtype eq 'NONE' || $feature_subtype eq 'host' && $has_host_genotypes || $feature_subtype eq 'pathogen' && $has_pathogen_genotypes) ? 'true' : 'false';
```

Comment:
The handling for the 'Pathogen-host interaction phenotype link' is inconsistent with the logic that we use here, because for metagenotype annotations, Canto treats the underlying concept (the metagenotype) as as interaction (like a genetic interaction). In this case, the link is always enabled, but you get a warning pop-up when clicking the link:

![image](https://user-images.githubusercontent.com/37659591/70429310-3a009f00-1a70-11ea-98bc-5137f4e27eef.png)


![image](https://user-images.githubusercontent.com/37659591/70429072-c6f72880-1a6f-11ea-9612-7056914140a3.png)

This message is a bit too generic for pathogen-host interactions, since the minimum required to create a metagenotype is _specifically_ one pathogen genotype (you can't create one if you just add host genotypes). Note that you don't need host genotypes because you can always use the default wild type host genotype on the Metagenotype Management page.